axis.go

```diff
@@ -20,8 +20,10 @@ const displayPrecision = 4
 
 // Ticker creates Ticks in a specified range
 type Ticker interface {
-	// Ticks returns Ticks in a specified range
-	Ticks(min, max float64) []Tick
+	// Ticks returns Ticks in a specified range and formatted according to the
+	// given format function.
+	// When no format is provided (nil) a sane default is used.
```

Comment:
`// When format is nil DefaultTickFormat is used.`

Sanity is subjective. We can claim this in talks, blogs and wiki articles, but I don't think it's appropriate in documentation.